src/components/drawer/FamilyMatchesDrawer.tsx

```diff
@@ -80,8 +72,7 @@ export const FamilyMatchesDrawer = ({ family }: TProps) => {
                   onClick={(e) => handleViewOtherDocsClick(e, document.document_slug)}
                   extraClasses="text-sm text-blue-600"
                 >
-                  View {document.document_passage_matches.length} {pluralise(document.document_passage_matches.length, "match", "matches")}{" "}
-                  highlighted in document
+                  View all matches highlighted in document
```

Comment:
We've lost the pluralisation here - is that intentional?

Reply:
Yes, unfortunately the data coming from the API is not reliable enough or accurate to display this. So we've defaulted to more general messaging.